packages/link/src/Link.ts

```diff
@@ -47,6 +47,22 @@ export class Link extends LikeAnchor(Focusable) {
         return this.anchorElement;
     }
 
+    constructor() {
+        super();
+        this.addEventListener('click', this.handleClickCapture, {
+            capture: true,
+        });
+    }
+
+    private handleClickCapture(event: Event): void | boolean {
+        if (this.disabled) {
+            event.preventDefault();
+            event.stopImmediatePropagation();
+            event.stopPropagation();
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events#none

Reply:
Yes this was one of the simplest ways. I feel there are many places in the codebase where we are adding the above code to handle event propagation.